lib/src/model.dart

```diff
@@ -377,22 +377,40 @@ class AnalysisResult {
 ///   - must not have a `publish_to` key.
 @JsonSerializable()
 class Repository {
+  final String provider;
```

Comment:
Would it make sense to represent this as an enum?

https://pub.dev/packages/json_serializable#enums